draft-lenders-dns-cbor.md

```diff
@@ -627,12 +555,100 @@ dns-response = [
 ~~~
 {:cddl #fig:dns-response title="DNS Response Definition"}
 
-# Further Compression with CBOR-packed {#sec:cbor-packed}
+# Compression with CBOR-packed {#sec:cbor-packed}
 
 If both DNS server and client support CBOR-packed {{-cbor-packed}}, it MAY be used for further
 compression in DNS responses.
 Especially IPv6 addresses, e.g., in AAAA resource records can benefit from straight referencing to
-compress common address prefixes.
+compress common address prefixes.[^6]{: mlenders}
+
+[^6]: Needs rewording.
+
+## Name Compression {#sec:name-compression}
+
+~~~ cddl
+Text-String-Suffix-Sequence-Packed-CBOR = #6.28259(rump)
+~~~
+
+For name compression, a new packing table setup tag TBD28259 ('n' and 'c' in ASCII) for CBOR-packed {{-cbor-packed}} is defined.
+It provides an implicit text string suffix sequence table for shared items _V_ which is prepended to the existing table for shared items that apply to the entire tag TBD28259 (by default empty table).
+The this implicit (i.e. not explicitly represented) table _V_ is constructed as follows:
+Any coherent sequence of text strings encountered within the rump of tag TBD28259, as well as any of its non-empty suffixes, are added to the table as an array in depth-first order.
```

Comment:
```suggestion
Any coherent sequence of text strings encountered within the rump of tag TBD28259, as well as any of its non-empty suffixes, are added to the table as arrays in depth-first order.
```

Reply:
Also, what exactly does "coherent sequence" mean? Is that a standing term?

Reply:
„CBOR sequence“ is a standing term. Cohererent is meant wrt text strings, so a (sub) sequence that only contains text strings.

Reply:
Then maybe "any CBOR sequence consisting of only text strings"? Maybe it's also just a personal English-word problem with "coherent".